invenio_query_parser/contrib/spires/parser.py

```diff
@@ -139,6 +139,18 @@ class SpiresSimpleQuery(UnaryRule):
     grammar = attr('op', [SpiresKeywordQuery, SpiresValueQuery])
 
 
+class RangeValue(LeafRule):
```

Comment:
I see your point. But at the same time, in the interest of avoiding code duplication, I think I would prefer to avoid duplicating things from the main parser in the spires contrib. There are some other imports in this file that is from the main parser as well I believe. (Although unfortunately it is hard to see right away due to `import *`).